config/routes.rb

```diff
@@ -3,4 +3,14 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-end
+
+  resources :amusement_parks, only: [:show] # get "/amusement_parks/:id", to: "amusement_parks#show"
+
+  resources :mechanics, only: [:show, :create] # get "/mechanics/:id", to: "mechanics#show"
```

Comment:
We don't need the `:create` on line 9 and since we aren't using it you should remove it. We want to avoid any unimplemented routes.